packages/clerk-js/src/ui/components/OrganizationProfile/CalloutWithAction.tsx

```diff
@@ -0,0 +1,47 @@
+import type { MouseEvent } from 'react';
+
+import type { LocalizationKey } from '../../customizables';
+import { Col, Flex, Link, Text } from '../../customizables';
+
+type CalloutWithActionProps = {
+  text: LocalizationKey;
+  actionLabel?: LocalizationKey;
+  onClick?: (e: MouseEvent<HTMLAnchorElement>) => Promise<any>;
+};
+export const CalloutWithAction = (props: CalloutWithActionProps) => {
```

Comment:
🔧 We should consider moving this to commons as the assumption is the same UI pattern will be used in multiple screens in the future.

Reply:
You have a point. Will update this